common/persistence/cassandraPersistence.go

```diff
@@ -1902,15 +1911,15 @@ GetFailureReasonLoop:
 	if rangeIDUnmatch {
 		return &ShardOwnershipLostError{
 			ShardID: d.shardID,
-			Msg: fmt.Sprintf("Failed to reset mutable state.  Request RangeID: %v, Actual RangeID: %v",
+			Msg: fmt.Sprintf("Failed to update mutable state.  Request RangeID: %v, Actual RangeID: %v",
 				requestRangeID, actualRangeID),
 		}
 	}
 
```

Comment:
plz separate the runIDUnmatch to here, and if not match, throw a new error such as currentRunIDMismatchError